web/react/components/create_comment.jsx

```diff
@@ -105,14 +106,17 @@ module.exports = React.createClass({
             this.lastTime = t;
         }
     },
-    handleUserInput: function(messageText) {
+    handleUserInput: function(message) {
+        var messageText = CreatePost.truncateText(message);
```

Comment:
While we are trying to get away from utils.jsx, I don't think it should be at the expense of creating possible dependancies as @crspeller pointed out. I think for this, it would be ok to add a function to utils.